bin/deepstream-start

```diff
@@ -70,3 +85,49 @@ if ( program.detach ) {
 	pidHelper.save( process.pid );
 
 }
+
+/**
+* Used by commander to parse the log level and and fails if invalid
+* value is passed in
+* @private
+*/
+function parseLogLevel( logLevel ) {
+	if( ! /debug|info|warn|error|off/i.test( logLevel ) ) {
+		console.error( 'Log level must be one of the following (debug|info|warn|error|off)' );
+		process.exit(1);
+	}
+	return logLevel.toUpperCase();
+}
+
+/**
+* Used by commander to parse the portnumber and fails if invalid
+* value is passed in
+* @private
+*/
+function parsePort( port ) {
+	const portNumber = Number( port );
+	if( !portNumber ) {
+		console.error( 'Provided port number must be an integer' );
+		process.exit(1);
+	}
+	return portNumber;
+}
+
+/**
+* Used by commander to parse the disabling/enabling of servers and fails if invalid
+* value is passed in
+* @private
+*/
+function parseServerEnabled( enabled ) {
```

Comment:
sounds like parseBoolean, I think the name is the second argument, so you could just write

```js
function parseServerEnabled( enabled, name ) {
  ...
  console.error( 'Invalid argument for ' + name + ', please provide true or false' );
}
```